### DISCUSSION TITLE
Double Quotes Inside Data

### DISCUSSION THREAD

**Discussion by TejendraPrasad** [JoshClose/CsvHelper#1996] [Q&A]
Hi All,

I have **First CSV** as below with double quotes

"Name","Description"
"test", " Desc"/1234"

I am trying to remove before Desc and also keep " in the field...

**My Output should be as below**

Name: test
Description: Desc"/1234
------------------------------------------------------------------------------------
I have **Second CSV** as below with no Double quotes

Name,Description
test,  Desc/1234

**My Output should be as below**

Name: test
Description: Desc/1234
------------------------------------------------------------------------------------
I tried below configuration but its removing double quotes in the middle...

```cs
 var config = new CsvConfiguration(CultureInfo.InvariantCulture)
            {
                HasHeaderRecord = true,
                BadDataFound = null,
                Delimiter = ",",
                //Quote = '"',
                Mode = CsvMode.NoEscape,
                TrimOptions = TrimOptions.Trim,
                //TrimOptions = TrimOptions.InsideQuotes                
            }; 
```

Please try to help with the configuration which works for both the csv file

**Comment by MiblaCz** [comment]
To escape quote inside a value you have to double it.
```
"Name","Description"
"test","Desc""/1234"
```

**Comment by ghost** [comment]
...or use the '\\' character depending on the context.

"Name","Description"
"test","Desc\\"/1234"
